### DISCUSSION TITLE
NixOS Help

### DISCUSSION THREAD

**Discussion by caiigee** [rafaelmardojai/firefox-gnome-theme#876] [Q&A] [answered]
Hi, is there anyone using NixOS here that somehow declaretively installed this theme via `home-manager`. If so, can you please share your configurations and ideas. Thanks!

**Comment by Z7G4N1U8** [comment, marked as answer]
now sure if you're still there but, you just need to take this repo as an input for flake and do flake = false, and then do this, another way is by using stylix which now support firefox gnome theme and also applies the colorscheme.
![image](https://github.com/user-attachments/assets/6a451f01-a23b-4073-bc7d-8335df60d238)

**Reply by caiigee** [reply]
Cool thank you! I did something similar with fetchFromGithub or fetchFromGit, I don't recall exactly which but the theme works for me now.
